msfrpc

```diff
@@ -41,7 +41,7 @@ opts = {
 }
 
 # Parse command line arguments.
-arguments.parse(ARGV) { |opt, idx, val|
+arguments.parse(ARGV) do |opt, idx, val|
```

Comment:
Personally, I don't really give a crap either way, but unless you have a test case that shows the precedence is an issue here, I think it should be idiomatic, which says use do/end if multiline